file-formats/aifp/examples/z_aff_example_aifp.aifp.json

```diff
@@ -0,0 +1,30 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "Example AIFP for ABAP File Format",
+    "originalLanguage": "en",
+    "abapLanguageVersion": "cloudDevelopment"
+  },
+  "generalInformation": {
+    "namespace": "AIFNS",
+    "aifCheck": "BP_EXIST"
+  },
+  "errorMessage": {},
```

Comment:
This should not be serialized into the example json because it has an initial value and is not a required field